app/app.go

```diff
@@ -370,12 +370,13 @@ func NewSommelierApp(
 
 	app.CellarFeesKeeper = cellarfeeskeeper.NewKeeper(
 		appCodec, keys[cellarfeestypes.StoreKey], app.GetSubspace(cellarfeestypes.ModuleName),
-		app.AccountKeeper, app.BankKeeper,
+		app.AccountKeeper, app.BankKeeper, app.CorkKeeper, app.GravityKeeper,
```

Comment:
So I think there are probably more than one situations like this where I actually fix the bug in the integration tests branch. Kind of awkward. Since this branch doesn't actually run any integration tests for cellarfees this inevitable nil reference error didn't show itself.

The integration tests branch uses this one as a base, so I could just implement the fixes here? It would also clean up that PR so it has fewer bug fixes in it.